lib/ae_mdw/db/mutations/oracle_extend_mutation.ex

```diff
@@ -6,14 +6,20 @@ defmodule AeMdw.Db.OracleExtendMutation do
   alias :aeser_api_encoder, as: Enc
   alias AeMdw.Blocks
   alias AeMdw.Db.Model
-  alias AeMdw.Db.Oracle
   alias AeMdw.Db.State
   alias AeMdw.Log
   alias AeMdw.Node.Db
   alias AeMdw.Txs
 
+  import AeMdw.Db.Oracle, only: [cache_through_read: 3]
+
+  import AeMdw.Db.Sync.Oracle,
+    only: [
+      cache_through_delete: 3,
+      cache_through_write: 3
+    ]
+
```

Comment:
Why use imports instead of regular aliases? I think we should stick to regular aliases since it makes it easier to find functions by grepping them

Reply:
Because the definition of the functions is well located from Oracle to Oracle module. 

But we can adopt always the alias expecting to find them with Module.<function> or Sync<Module>.<function>